Lesson 22/index.js

```diff
@@ -0,0 +1,15 @@
+"use script";
+
+alert("Enter two numbers to see if the first is greater than the second");
+
+let firstNumber = prompt("enter the first number", '');
```

Comment:
Make sure to convert the prompt inputs to numbers before comparing them. What is happening here is a string comparison